src/components/carousel/children/arrows.jsx

```diff
@@ -0,0 +1,17 @@
+export const NextArrow = (props) => {
+	const { className, onClick } = props;
+	return (
+		<button type="button" title="Next" data-role="none" className={className} onClick={onClick}>
+			&#62;
```

Comment:
is there a reason that we can't use the ">" or "<" here?

Reply:
You guys are reviewing a WIP!

Reply:
@JackHowa : Confirm w/ @visionbegin but I don't think we're going to need this file. Good question, though!

Reply:
ok sounds like this is going to be removed